lldb/source/Plugins/LanguageRuntime/Swift/ReflectionContextInterface.h

```diff
@@ -145,6 +146,23 @@ class ReflectionContextInterface {
       swift::reflection::DescriptorFinder *descriptor_finder) = 0;
   virtual swift::remote::RemoteAbsolutePointer
   StripSignedPointer(swift::remote::RemoteAbsolutePointer pointer) = 0;
+  struct AsyncTaskInfo {
+    bool isChildTask;
```

Comment:
any benefit in having them all be = false?

Reply:
do you mean initializing them all to false? I don't know if there's a benefit, but it's fine with me.